sdk/lib/src/main/java/com/friendlycaptcha/android/sdk/FriendlyCaptchaWidgetHandle.kt

```diff
@@ -70,7 +70,7 @@ class FriendlyCaptchaWidgetHandle(
     private var startPending = false
 
 
-    private val url: String = "file:///android_asset/index.html?" +
+    private val url: String = "file:///android_asset/friendlycaptcha/index.html?" +
```

Comment:
I think we need to rename the file itself and change it here, not add it to the  directory